modules/null/federation/main.tf

```diff
@@ -7,7 +7,8 @@ locals {
   federate_hub_cmds = templatefile("${path.module}/federate_hub.tpl", {
     ssh_key_path = var.hub_info.hub_private_ssh_key_path
     dsf_gw_ip    = var.gw_info.gw_ip_address
-    dsf_hub_ip   = var.hub_info.hub_ip_address
+    # null check to allow destroy in case we got: Error creating EIP: AddressLimitExceeded: The maximum number of addresses has been reached.
```

Comment:
I don't like this fix. It's really confusing for the normal user like those values are valid. I also don't think this is a real solution. The EIP error can attach the user from all kinds of places. IAM role quota, vpc quota, any other quota. I wouldn't want us to pass nulls and empty strings in all the places.

  I think we should open a bug for this and assign someone to solve the quota issue specifiicaly

Reply:
I will revert this, but it means that if and when it happens to us again, we won't be able to destroy (in case I want to move to another region because I can't delete the existing EIPs) unless we termporarily add these changes.
We talked about the quota issues in our last sync and Assaf opened 2 tasks for it.